### PR TITLE
Adding constant declarations

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
Adding constant declarations suggested by @koshy-thomas. Thanks Koshy.

Note: This is related to PR 205.

### CODE REVIEW

File: docs/sdks/create-api-key.md
Comment (koshy-thomas) [inline docs/sdks/create-api-key.md:34]: One doubt here since this is a create apikey example shouldn't we assume that the user doesn't have an apikey yet? Hence shouldn't the C8client login method used here be based on email and password rather than apikey for "creating apikey" example? Any thoughts on this?

Comment (diana-macrometa) [reply docs/sdks/create-api-key.md:34]: Yes. Assume that the user is creating an API key because they do not have one. Or we could just point them at the authentication topic and let them decide how to log in.

Comment (jesusgonzalez12) [reply docs/sdks/create-api-key.md:34]: ok, thanks guys.

Suggestions have been applied.

**Comment by jesusgonzalez12** [comment]
Hello team.
Please review again, approve and merge.

**Comment by koshy-thomas** [comment]
> Hello team. Please review again, approve and merge.

@diana-macrometa @jamesm-macrometa can anyone of you review once more and approve if everything's fine

**Review by diana-macrometa — APPROVED** [review]
LGTM!

**Pull request merged**
